tensorflow/compiler/jit/deadness_analysis.cc

```diff
@@ -1100,11 +1318,11 @@ Status DeadnessAnalysisImpl::PopulateWithReversePostOrder(
     }
   }
 
-  for (Node* n : rpo) {
+  for (Node* n : tpo) {
     // The nodes added to should_revisit in the previous loop need to be
     // revisited now.  Reprocesing these initial nodes may add *their* consumers
     // to should_revisit, and these newly added nodes will also be processed by
-    // this very same loop.  Since we're traversing the graph in reverse post
```

Comment:
I checked the wiki page: https://en.wikipedia.org/wiki/Topological_sorting

My read is that although rpo is (certainly) a valid topological order, it is just a kind of topological order. In other words, there are many kinds of topological orders. So, it would be more precise to say it is a topological order instead of rpo.

I added some more comments before the procedure and give a ref to the wiki page. Please let me know what you think.

Reply:
I see.  On second thought I agree that it is better to call this a topological sort since we're no longer doing a DFS to compute the order.